alpha-build-machinery/make/examples/multiple-binaries/Makefile

```diff
@@ -1,3 +1,22 @@
 include $(addprefix ../../, \
 	golang.mk \
 )
+
+CROSS_BUILD_BINDIR :=_bin
+
+cross-build-darwin-amd64:
+	+@GOOS=darwin GOARCH=amd64 $(MAKE) --no-print-directory build GO_BUILD_BINDIR:=$(CROSS_BUILD_BINDIR)/darwin_amd64
```

Comment:
what is the `+`?

Reply:
always run. (iow. actually run on dry-run, $(MAKE) should imply it but this is explicit)

Reply:
learned something new :+1: